plasmapy/__init__.py

```diff
@@ -28,14 +28,7 @@
 # ----------------------------------------------------------------------------
 import pkg_resources
 
-from plasmapy import (
-    diagnostics,
-    formulary,
-    particles,
-    plasma,
-    simulation,
-    utils,
-)
+from plasmapy import diagnostics, formulary, particles, plasma, simulation, utils
```

Comment:
This looks a bit weird but I guess it's just below the line length limit, and I'm not going to argue with autotools on that point.

Reply:
Agreed! I guess it's practice in bowing down to our robotic overlords.